src/Simplex/Chat/Types.hs

```diff
@@ -1171,6 +1182,7 @@ fileTransferCancelled (FTSnd FileTransferMeta {cancelled} _) = cancelled
 fileTransferCancelled (FTRcv RcvFileTransfer {cancelled}) = cancelled
 
 -- For XFTP file transfers FSConnected means "uploaded to XFTP relays"
+-- Local files are always FSComplete
```

Comment:
FileStatus only relates to file transfers I think (maybe even only SMP file transfers? not sure)
```suggestion
```